blog/_posts/2019-11-13-affiliations-feature.md

```diff
@@ -0,0 +1,67 @@
+
+Using one of this blog’s coauthors, searching for first_author:“Templeton, Matthew” brings up a list of his papers along with a few other “Matthew Templeton”s who are active researchers in ADS-relevant fields.  Selecting the “astronomy” collection and “limit to” yields (mostly) just his set of first-authored publications, and refereed lowers it further to a manageable number.  If you then open the Affiliations facet, you’re presented with a list of all affiliations (of any author) contained within my papers.  Matthew Templeton’s historical affiliations -- AAVSO, New Mexico State University, Yale University, Los Alamos National Laboratory -- are among the most common in the list.  However, you also find affiliations of co-authors too, for example Iowa State and the Center for Astrophysics.
+
+Affiliations in the ADS have been indexed in several different fields, with the intention of allowing multiple use cases.  We currently have assigned identifiers with parent/child relationships where a child may have multiple parents.  This has required a few modifications to remain useful. For instance, so that University of California schools can identify departments, we have assigned them a parent status, even though the "University of California System" should really be the parent level.  Likewise, NASA's Goddard Space Flight Center is at a parent level, as are France's CNRS institutions to allow for further subdivision.  Further work on a schema to allow more complex relationships between institutions is under development in conjunction with work by the ROR Community (ror.org).
+
+The solr fields in ADS that store the affiliation information are:
+
+* aff: raw affiliation string, tokenized
+
+* aff_id: a string containing one or more of the affiliation ids listed in the parent_child.tsv file in column "ADS_ids."  This field will soon also accept 9-digit ROR ids.
+
+* inst: the abbreviated institution name (e.g. "U Adelaide") listed in the parent_child.tsv file
+
+So, you could search for “aff:Harvard” and get back all affiliations that contain Harvard in the affiliation string - including “Harvard Street”.  To ensure you get only the University, you could search for “aff:Harvard University”, but that would return only affiliations strings with that exact phrase excluding “Harvard Univ.”.  Better would be to use the identifier for Harvard University by searching “aff_id::A00211”.  This returns all affiliations which contain Harvard University at the parent level.  Best yet would be to search by institution, “inst:”Harvard U” which returns affiliations which contain Harvard University at the parent level along with those at the child level.
```

Comment:
* "Better would be to use the identifier for Harvard University by searching “aff_id::A00211”.  This returns all affiliations which contain Harvard University at the parent level." Is this true? It won't find matches with that ID in the child level?
* There's some weirdness going on in this paragraph with quotes and other punctuation around the search strings. I'm guessing `“aff:Harvard University”` should actually be `aff:"Harvard University”`, `“aff_id::A00211”` should be `“aff_id:A00211”`, and `“inst:”Harvard U”` should be `inst:”Harvard U”`. If you're trying to show that this is a searchable string by including the extra quotes around the fielded search term, maybe make each of those three a hyperlink to the actual search - this would be a useful demo for users too.

Reply:
point 1  - correct.  That's why inst is so much more powerful.

Reply:
Oh,  A00211 is never at the child level.